### PR TITLE
Add extra testcases

### PR DESCRIPTION
**Pull request by salnad**
Add extra test cases to FindMeetingQueryTest.java to handle edge cases with optional attendees.

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 1]
```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithInvalOptional() {
+    // Have each person have different events. We should see two options because each person has
```
Comment (pruess-google) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:137]: Looks like your lines are over the character limit here. Would you mind wrapping them? The same goes for some other comments in this file

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 2]
```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithInvalOptional() {
```
Comment (pruess-google) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:136]: What does "Inval" mean here? Is it "Invalid"? In general it's better to not abbreviate any words in method names to avoid ambiguity. Would you mind updating this?

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 3]
```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
...
+        Arrays.asList(
+            TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0830AM, TIME_0900AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void everyAttendeeIsConsideredWithValOptional() {
```
Comment (pruess-google) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:174]: Same here, I'm not sure what "Val" means

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 4]
```diff
@@ -130,6 +132,80 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithInvalOptional() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times. Add optional person C who takes up whole day (therefore, not
+    // considered)
+    //
+    // Events  :       |--A--|     |--B--|
```
Comment (pruess-google) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:141]: Nice visualization of the test cases :)

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java (hunk 1 above)
Comment (salnad) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:137]: The auto formatter didn't have issue with this specific comment. I run it and it cleaned some of the comments up but not this one. Could there be an issue with my formatter?

Comment (pruess-google) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:137]: Hmm I wonder if there's a setting where you can set the maximum line limit? It's also possible the formatter doesn't try to modify comments at all: I'm not sure all of them do so, so you may have to manually make sure comments aren't over the line limit.
